gsutil.py

```diff
@@ -16,14 +16,26 @@
 
 """Wrapper module for running gslib.__main__.main() from the command line."""
 
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+
 import os
 import sys
 import warnings
 
 # TODO: gsutil-beta: Distribute a pylint rc file.
 
-if not (2, 6) <= sys.version_info[:3] < (3,):
-  sys.exit('gsutil requires python 2.6 or 2.7.')
+if not ((2, 6) <= sys.version_info[:3] < (3,)
```

Comment:
We only need `version_info[:2]` for both of these checks, right? Since we're only concerned with major and minor version, not patch version

Reply:
sounds good, I'll clean this up maybe into a function in utils or somewhere since `__init__.py` does the same check

Reply:
Logic revised. I can add to utils or something if that would be a better place, and then just call it at this stage and at __init__.py

Reply:
See the most recent two commits and the comment on gslib/__init__.py; thoughts on adding a top level __init__.